pkg/lib/bundle/testdata/validate/valid_bundle_additional_objects/manifests/etcdrestores.etcd.database.coreos.com.crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
What do you think about adding these manifests to existing valid bundles instead of creating a new directory for them?

Reply:
+1

Reply:
Makes sense. I was being cautious with the idea "not to modify fixtures that existing tests rely on" but it's probably not relevant in this simple case.